python/lib/cloudutils/serviceConfig.py

```diff
@@ -54,7 +54,7 @@ def configration(self):
         except:
             logging.debug(formatExceptionInfo())
             if self.syscfg.env.mode == "Server":
-                raise CloudRuntimeException("Configure %s failed, Please check the /var/log/cloudstack/setupManagement.log for detail"%self.serviceName)
+                raise CloudRuntimeException("Configure %s failed, Please check the /var/log/cloudstack/management/setupManagement.log for detail"%self.serviceName)
```

Comment:
Does this need to be hard-coded? I had a quick look into the python logging and am hoping we can retrieve the path from there. just a thought